pkgs/development/python-modules/pypass/default.nix

```diff
@@ -0,0 +1,89 @@
+{ buildPythonPackage
+, click
+, colorama
+, enum34
+, fetchFromGitHub
```

Comment:
Not used, as well as fetchgit.

Reply:
Thanks! Would be nice to have editor support to point these out.. Or `nixpkgs-fmt` could point these out too. :thinking:

Reply:
Fixed.